team_wizz.js

```diff
@@ -1 +1,33 @@
 // Fetch all routes for the airline and insert into #wizz
+
+(async () => {
+  const firstFlight = await fetch("https://flyapi.onrender.com/tlv_ber/wizz");
+  const secondFlight = await fetch("https://flyapi.onrender.com/arn_lhr/wizz");
```

Comment:
Here you have 3 fetches, but what if you need to do 100+ fetches? do you want to hard code the fetches?
I don't think so - be lazy! create a function like `getRoutePrice(...)` so you could do all the work in a generalized manner.

Reply:
Created getRouteRice function